### PR TITLE
Feature/issue 158 - Use Lambda role for S3 connection

### PR DESCRIPTION
**Pull request by torimcd**
Github Issue: #158 Use Lambda role instead of services EDL for S3 connection

### Description

Using the services EDL results in permissions issues when loading granules in SIT & UAT. Removing the EDL parameters and letting the S3 connection use the lambda role resolves the issue.

### Overview of work done

Removed the access parameters on the S3 resource in the connections module, removed the /s3credentials endpoint constants and EDL parameters from iam policies

### Overview of verification done

No changes to unit tests as this only affects the deployed environment.

### Overview of integration done

Deployed to SIT and verified granules can be loaded by CNM lambda on ingest to cumulus SIT venue.

## PR checklist:

* [x] Linted
* [ ] Updated unit tests
* [x] Updated changelog
* [x] Integration testing

_See [Pull Request Review Checklist](../CONTRIBUTING.md#reviewing) for pointers on reviewing this pull request_

### CODE REVIEW

**Review by frankinspace — APPROVED** [review]
Looks good to me. Simplifies the data access quite a bit.

File: terraform/hydrocron-lambda.tf
Comment (nikki-t) [inline terraform/hydrocron-lambda.tf:1]: I think we can also remove the following from `main.tf`: https://github.com/podaac/hydrocron/blob/3ade1bbd46c5cc095a5adff8f00928320967b83d/terraform/main.tf#L49-L55

**Review by nikki-t — APPROVED** [review]
This looks good to me. At some point we may want to clean up our use of the SSM parameter store as I think it is only being used to store the DynamoDB endpoint URL and API URL. I am also not sure how the EDL credentials were being defined and stored as I don't see them in the Terraform.

File: terraform/hydrocron-lambda.tf
Comment (torimcd) [reply terraform/hydrocron-lambda.tf:1]: good catch, completely missed those

**Pull request merged**